dashboard/src/main/resources/templates/dashboard.ftl

```diff
@@ -40,8 +41,26 @@
       </div>
     </section>
     
+    <section class="piecharts">
+      <h3>Linkage Errors</h3>
+      
+      <#assign ratio = linkageErrorCount / totalArtifacts >
+      <#assign endPointX = pieChart.calculateEndPointX(100, 100, 100, ratio)>
+      <#assign endPointY = pieChart.calculateEndPointY(100, 100, 100, ratio)>
+      
+      <p>${linkageErrorCount} out of ${totalArtifacts} artifacts have linkage errors.</p>
```

Comment:
Subject-verb agreement when `linkageErrorCount` is 1.

"1 out of 187 artifacts have linkage errors." -> "1 out of 187 artifacts has linkage errors."

Reply:
done